src/instance.rs

```diff
@@ -350,17 +346,40 @@ where
 {
     /// Create new instance of T and move it under python management
     /// Returns `Py<T>`.
-    pub fn new<F>(py: Python, f: F) -> PyResult<Py<T>>
+    pub fn new(py: Python, value: T) -> PyResult<Py<T>>
     where
-        F: FnOnce() -> T,
         T: PyTypeObject + PyTypeInfo,
     {
         let ob = <T as PyTypeCreate>::create(py)?;
-        ob.init(f)?;
+        ob.init(value)?;
 
         let ob = unsafe { Py::from_owned_ptr(ob.into_ptr()) };
         Ok(ob)
     }
+
+    /// Create new instance of `T` and move it under python management.
+    /// Returns references to `T`
+    pub fn new_ref(py: Python, value: T) -> PyResult<&T>
```

Comment:
We don't need these two functions, since now `PyRef::new` and `PyRefMut::new` exist